WordPress/gutenberg
# Hide "Create template part" menu from block settings dropdown for classic themes

## What?

I noticed that in classic themes, template parts can be created unintentionally in certain scenarios:

- Post Editor > Create a synced pattern > Edit original > Select a block within a synced pattern > Open the block settings dropdown
- Apperance > Patterns > My Patterns > Select a block within a pattern  > Open the block settings dropdown

![image](https://github.com/user-attachments/assets/30a70629-a59f-4bf2-b479-226af368f0ec)

## Why?

Classic themes don't allow us to insert template parts via the post editor or site editor, so we should always hide this menu.

## How?

Add `isBlockTheme` check.

## Testing Instructions

- Activate a classic theme.
- The "Create template part" menu should not be displayed in the following locations:
  - Post Editor > Create a synced pattern > Edit original > Select a block within a synced pattern > Open the block settings dropdown
  - Apperance > Patterns > My Patterns > Select a block within a pattern  > Open the block settings dropdown
